ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -68,6 +71,13 @@ StackLayout {
         txModalLoader.close()
     }
 
+    Connections {
+        target: profileModel
+        onContactListChanged: {
+            console.log("Contact list changed");
```

Comment:
```suggestion
```

Reply:
Damn I should pay better attention to these haha